eclair-core/src/test/scala/fr/acinq/eclair/payment/RelayerSpec.scala

```diff
@@ -416,4 +411,11 @@ class RelayerSpec extends TestkitBaseClass {
     assert(fwd.channelId === origin.originChannelId)
     assert(fwd.message.id === origin.originHtlcId)
   }
+
+  test("get usable balances") { f =>
+    import f._
+    val sender = TestProbe()
+    sender.send(relayer, GetUsableBalances)
```

Comment:
Come on, this test is meaningless, the list is empty.

At the very least (and this doesn't even test e.g. inversion of send/receive amounts):

```scala
  test("get usable balances") { f =>
    import f._
    val sender = TestProbe()
    relayer ! LocalChannelUpdate(null, channelId_ab, channelUpdate_ab.shortChannelId, a , None, channelUpdate_ab, makeCommitments(channelId_ab))
    relayer ! LocalChannelUpdate(null, channelId_bc, channelUpdate_bc.shortChannelId, c, None, channelUpdate_bc, makeCommitments(channelId_bc))
    sender.send(relayer, GetUsableBalances)
    assert(sender.expectMsgType[Iterable[UsableBalances]].size === 2)
  }
```

Reply:
Perhaps I should revise my notion of really simple test :)